docs/diagrams/LogicClassDiagram.puml

```diff
@@ -6,7 +6,7 @@ skinparam classBackgroundColor LOGIC_COLOR
 
 package Logic {
 
-Class AddressBookParser
+Class NuSchedulerParser
 Class XYZCommand
```

Comment:
there should be 2 classes: ZX commands and Y commands that inherit from Command

Reply:
Thanks for the catches!